extensions/amp-analytics/amp-analytics.md

```diff
@@ -156,6 +156,11 @@ Type attribute value: `gemius`
 
 Adds support for Gemius Audience/Prism analytics. Additionally, the gemius-provided `prefix` and `identifier` variables must be specified. It's also possible to specify the optional variable `extraparams` (key1=value1|key2=value2). More details can be found at [gemius.com](https://www.gemius.com).
 
+### Google Adwords
```

Comment:
nit: In all occurrences of "Adwords", capitalize the "W" so it reflects the product name.